webapp/store/logic.py

```diff
@@ -322,6 +323,21 @@ def get_categories(categories_json):
         i = i + 1
 
 
+def get_docs_topic_id(metadata_yaml):
+    """
+    Return discourse topic ID or None
+    """
+    base_url = os.environ["DISCOURSE_URL"]
```

Comment:
we won't have other discourses
```suggestion
    base_url = "https://discourse.juju.is/"
```